README.md

```diff
@@ -123,25 +133,53 @@ The important features are:
 
 ### 4. Run the pipeline 
 
-The most basic run command for this pipeline is: 
+The most basic run command for this pipeline with all required paramters looks like this: 
 
 ```
-nextflow run main.nf --input samples.csv --ref reference.fasta  --intervalList_path path_to_intervals --ponvcf pon.vcf.gz
-        
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants
 ```
-**Note**: Please use the command provided in the script `scripts/run_pipeline_on_gadi_script.sh` on NCI Gadi HPC.
 
 By default, this will generate `work` directory, `results` output directory and a `runInfo` run metrics directory inside the results directory. 
 
+
+This pipeline has been tested for NCI Gadi HPC and the paramaters `--whoami` and `--gadi_account` will have to be included as discussed in the section [Infrastructure usage and recommendations](#infrastructure-usage-and-recommendations) .
+```
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants \
+                     -profile gadi \
+                     --whoami $(whoami) --gadi_account $PROJECT    
+```
```

Comment:
Suggest moving this to [Infrastructure usage and recommendations](#infrastructure-usage-and-recommendations) to reduce repetition